astropy/modeling/core.py

```diff
@@ -899,6 +899,12 @@ def bounding_box(self, limits):
     def evaluate(self, *args, **kwargs):
         """Evaluate the model on some input variables."""
 
+    def render(self, *args, **kwargs):
+        """
```

Comment:
@adonath: bbox evaluation directly in the  `evaluate` method doesn't make much sense on its own.

I think there should be a `render()` method and there should be no separate `render_model` function at all.  In other words, just move all the code that currently exists as `render_model` into `Model.render`.